src/main/java/duke/exceptions/DukeArrayOutOfBoundException.java

```diff
@@ -0,0 +1,8 @@
+package duke.exceptions;
+
+public class DukeArrayOutOfBoundException extends DukeException{
+
+    public DukeArrayOutOfBoundException() {
+        super("Index cannot be zero or negative");
+    }
+}
```

Comment:
I like how you created more exception classes for `DukeException`, makes exceptions a lot clearer in the code!